### DISCUSSION TITLE
New to coding

### DISCUSSION THREAD

**Discussion by Thyme-spin** [skills/.github#746] [Code with Copilot]
I am new to coding, and will be working with some friends. Does anybody have some tips, tricks, or ideas? Our project is a secret, so no further information can be given. Computer tech is not really my forte, but I'm looking forward to learning this. Also, when I say it's not my forte, I really mean that it's not.

**Comment by Thyme-spin** [comment]
I don't have discord yet, but if/when I do decide to get it, I'll follow
you.

**Reply by Moluklin** [reply]
I am new here and I want to learn
